.github/actions/run-fixtures-test/action.yml

```diff
@@ -0,0 +1,52 @@
+name: Run a fixtures test
+description: Run a fixtures test
+
+inputs:
+  fixtures-dir:
+    description: Path to the fixtures directory
+    required: true
+  command:
+    description: The git-cliff command to run
+    required: false
+    default: ""
+
+runs:
+  using: composite
```

Comment:
_comment:_ Wow! I didn't know this was a thing. Very clean 💯

Reply:
Yes, I learned it recently. :smile: 

FYI, there is a similar way to reuse jobs, called Reusable Workflows.
However, it has a bug in the local reference feature, I didn't use it here.
https://github.community/t/ref-head-in-reusable-workflows/203690/72

Reply:
Thanks for sharing (and also implementing `composite`)!